test/HostnameTest.php

```diff
@@ -506,6 +506,25 @@ public function testAdditionalUTF8TLDs()
         }
     }
 
+    public function testAdditionalPunycodedTLDs()
+    {
+        $validator = new Hostname(Hostname::ALLOW_ALL);
+        // Check UTF-8 TLD matching
+        $valuesExpected = [
```

Comment:
I'll appreciate if you can do it testAdditionalUTF8TLDs as well.

What I miss with the descriptions I don't quickly understand why that case is significative.

```
return [
        'UTF8 TLD' => 'test123.онлайн',
        'Punyencoded TLD' => 'test123.xn--80asehdb',
...
    ];
```